### PR TITLE
Add Elk-M1

### PR DESCRIPTION
**Pull request by gwww**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):**
https://github.com/home-assistant/home-assistant/pull/16952

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/alarm_control_panel.elkm1.markdown [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+layout: page
+title: "Elk-M1 Alarm Control Panel"
+description: "Instructions how to integrate Elk M1 alarm control panel."
+date: 2018-10-07 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: elkproducts.png
+ha_release: '0.80'
```
Comment (MartinHjelmare) [inline source/_components/alarm_control_panel.elkm1.markdown]: 0.81

File: source/_components/alarm_control_panel.elkm1.markdown [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+ha_category: Alarm
+ha_iot_class: "Local Push"
+---
+
+The `elkm1` platform allows you to control your [Elk-M1](https://www.elkproducts.com/m1_controls.html).
+
+A Elk-M1 area (also known as partition) is represented as an `alarm_control_panel`.
```
Comment (MartinHjelmare) [inline source/_components/alarm_control_panel.elkm1.markdown]: An Elk-M1 area ...

File: source/_components/elkm1.markdown [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+layout: page
+title: "Elk-M1 Controller"
+description: "Instructions to setup the Elk-M1 controller."
+date: 2018-10-07 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: elkproducts.png
+ha_release: '0.80'
```
Comment (MartinHjelmare) [inline source/_components/elkm1.markdown]: 0.81

File: source/_components/elkm1.markdown [hunk 4]
```diff
@@ -0,0 +1,253 @@
...
+      description: List to exclude in the form of either `<value>` or `<value>-<value>` where `<value>` is a number or a X10 housecode. See configuration below for examples of ranges.
+      type: list
+      required: false
+      default: None excluded.
+{% endconfiguration %}
+
+Example configuration of the above:
+```yaml
+elkm1:
+host: elks://IP_ADDRESS
```
Comment (MartinHjelmare) [inline source/_components/elkm1.markdown]: Indentation is off.

**Review by MartinHjelmare — APPROVED** [review]
Looks good! Can be merged when the build passes.

**Comment by gwww** [comment]
Thank you!

**Pull request merged**